src/VisualStudio/TestUtilities.Next/VisualStudioTestUtilities.Next.csproj

```diff
@@ -9,7 +9,6 @@
     <AssemblyName>Roslyn.VisualStudio.Test.Utilities.Next</AssemblyName>
     <PlatformTarget>x86</PlatformTarget>
     <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
-    <CopyNuGetImplementations>false</CopyNuGetImplementations>
```

Comment:
I think we still want this to be false, otherwise we're copying in all sorts of stuff?